### PR TITLE
Turso03 makefile updates

### PR DESCRIPTION
**Pull request by ykempf**
Please someone report successful building and running to check that permissions are OK for other users than me.

### CODE REVIEW

File: MAKE/Makefile.carrington_gcc_openmpi [hunk 1]
```diff
@@ -11,20 +11,21 @@
 # module load gnu9/9.3.0
 # module load openmpi4/4.0.5
 # module load pmix/3.1.4
-
+# Oneliner for command line:
+# module purge ; module load gnu9/9.3.0 ; module load openmpi4/4.0.5 ; module load pmix/3.1.4
```
Comment (markusbattarbee) [inline MAKE/Makefile.carrington_gcc_openmpi:15]: Add: 
remember to compile inside interactive session:
/usr/bin/srun -M carrington --interactive -n 16 --mem=4G --time=00:20:00 -p short --pty bash

File: MAKE/Makefile.vorna_gcc [hunk 2]
```diff
@@ -1,9 +1,11 @@
...
 #
 # Reccomended modules in .bashrc:
-# module load GCCcore/8.3.0
-# module load OpenMPI/3.1.4-GCC-8.3.0
-# module load Eigen
+# module purge
+# module load GCC/10.2.0
+# module load OpenMPI/4.0.5-GCC-10.2.0
+# Same in one line to past on the command line:
+# module purge ; module load GCC/10.2.0 ; module load OpenMPI/4.0.5-GCC-10.2.0
```
Comment (markusbattarbee) [inline MAKE/Makefile.vorna_gcc:9]: Add:
remember to compile inside interactive session:
/usr/bin/srun -M vorna --interactive -n 16 --mem=4G --time=00:20:00 -p test --pty bash

File: MAKE/Makefile.vorna_gcc [hunk 3]
```diff
@@ -57,38 +59,36 @@ testpackage: CXXFLAGS += -DUSE_JEMALLOC -DJEMALLOC_NO_DEMANGLE
...
 
 #header libraries
-INC_FSGRID = -I$(LIBRARY_PREFIX)/fsgrid/
-INC_DCCRG = -I$(LIBRARY_PREFIX)/dccrg/
-INC_VECTORCLASS = -I$(LIBRARY_PREFIX)/vectorclass/
-
+INC_FSGRID = -I/proj/group/spacephysics/libraries/fsgrid/
+INC_DCCRG = -I/proj/group/spacephysics/libraries/dccrg/
+INC_VECTORCLASS = -I/proj/group/spacephysics/libraries/vectorclass/
+INC_EIGEN = -I/proj/group/spacephysics/libraries/
```
Comment (markusbattarbee) [inline MAKE/Makefile.vorna_gcc:92]: I think these should use the LIBRARY_PREFIX variable

**Review by markusbattarbee — CHANGES_REQUESTED** [review]
Was able to compile and run with both of these. I'd also recommend purging all old vorna and carrington makefiles in this PR.

File: MAKE/Makefile.carrington_gcc_openmpi (hunk 1 above)
Comment (ykempf) [reply MAKE/Makefile.carrington_gcc_openmpi:15]: Actually there the "old" interactive call still works. But good reminder. Shall I instead add "remember to compile in an interactive session (see wiki for current best practices)"?

**Comment by ykempf** [comment]
So shall I nuke MAKE/Makefile.vorna_intel too or do you/someone want to test an intel version too?

File: MAKE/Makefile.carrington_gcc_openmpi (hunk 1 above)
Comment (markusbattarbee) [reply MAKE/Makefile.carrington_gcc_openmpi:15]: sounds good (and make sure the wiki is up to date :) )

**Comment by markusbattarbee** [comment]
> So shall I nuke MAKE/Makefile.vorna_intel too or do you/someone want to test an intel version too?

probably not worth the hassle. If we need to, it should be a clean start anyway as it's a completely new OS release.

**Comment by markusbattarbee** [comment]
right, so I see this has the 2D shells VDF output in it - noteworthy that it is not the same behaviour as in earlier 3D runs, as instead of having the kinda-arbitrary-but-simple cellID stride, it also for 3D runs calculates the dominant coordinate and uses it as the stride.

**Comment by ykempf** [comment]
Oh no. I didn't notice the cherry-pick made it in.

File: MAKE/Makefile.vorna_gcc (hunk 2 above)
Comment (ykempf) [reply MAKE/Makefile.vorna_gcc:12]: Done

File: MAKE/Makefile.carrington_gcc_openmpi (hunk 1 above)
Comment (ykempf) [reply MAKE/Makefile.carrington_gcc_openmpi:15]: Juha confirmed that interactive is deprecated also on Carrington.

**Pull request merged**